n3fit/src/n3fit/model_gen.py

```diff
@@ -396,29 +399,29 @@ def output_layer(ilayer):
 
 
 def pdfNN_layer_generator(
-    inp=2,
-    nodes=None,
-    activations=None,
-    initializer_name="glorot_normal",
-    layer_type="dense",
-    flav_info=None,
+    inp: int = 2,
+    nodes: List[int] = None,
+    activations: List[str] = None,
+    initializer_name: str = "glorot_normal",
+    layer_type: str = "dense",
+    flav_info: dict = None,
     fitbasis="NN31IC",
-    out=14,
-    seed=None,
-    dropout=0.0,
+    out: int = 14,
+    seed: int = None,
+    dropout: float = 0.0,
     regularizer=None,
     regularizer_args=None,
-    impose_sumrule=None,
+    impose_sumrule: str = None,
     scaler=None,
-    parallel_models=1,
+    parallel_models: int = 1,
```

Comment:
Same as above, please complete the annotations.

Reply:
Done except for the photons argument [here](https://github.com/NNPDF/nnpdf/blob/38a1ccbcf1cbfa41540e6ceef13c9d96acdc117f/n3fit/src/n3fit/model_gen.py#L416) , which I didn't find documented anywhere

Reply:
@niclaurenti what is the object type for the photon?

Reply:
Yes exactly. It is the object created in the `validphys.photon.compute.Photon` class